packages/config/config/devices/0x0271/motionswitch_led.json

```diff
@@ -75,13 +75,40 @@
 			"defaultValue": 1
 		},
 		{
-			"#": "9",
-			"label": "SLAVE_MODE",
-			"description": "Disable local control",
+			"#": "9[0x01]",
+			"label": "Gateway Check",
```

Comment:
@nechry Do you have a manual which shows that this one has no "External Control" mode? I only found [this](http://manuals-backend.z-wave.info/make.php?lang=en&sku=40111665101&cert=ZC10-18076193) (near the bottom) and it looks like param 9 is the same as for the other devices.

Reply:
I found this: [one](https://manuals-backend.z-wave.info/make.php?lang=en&sku=STEEMSWITCH&cert=ZC10-18076193) This is the only model I don't own.
your document seems more recent to me, so I prefer to play safe and add the bitset.

Reply:
Thanks, I'll go with the bitset then. Even on the one you linked it looks a bit like it is supposed to be a bitset but they entered the values weirdly.

Reply:
in fact like bit 2 (LED is permanently on) override bit 0 and 1 it is true that we could simplify with a list. Again I relied on the initial config for the refactoring.

Reply:
@AlCalzone do you prefer I convert to list with selection:

```json
{
  "#": "9",
  "label": "Slave mode",
  "description": "",
  "valueSize": 1,
  "minValue": 0,
  "maxValue": 4,
  "defaultValue": 2,
  "unsigned": true,
  "allowManualEntry": false,
  "options": [
  {
	  "label": "Normal mode",
	  "value": 0
  },
  {
	  "label": "Slave mode",
	  "value": 1
  },
  {
	  "label": "Normal mode with checking lifeline",
	  "value": 2
  },
  {
	  "label": "Slave mode with checking lifeline",
	  "value": 3
  },
  {
	  "label": "LED is permanently on",
	  "value": 4
  }
  ]
}
```

Reply:
I reworked most of the files to templates, let me just copy that. It's probably easier to understand than the bitset.

Reply:
great, yes at the end a list even if behind it's bit sets it's even more user-friendly